src/pages/Compare/index.spec.tsx

```diff
@@ -384,7 +383,12 @@ describe(__filename, () => {
       store,
     });
 
-    expect(root.find(`.${styles.error}`)).toHaveLength(1);
+    expect(
+      root
+        .find(VersionFileViewer)
+        .children()
+        .text(),
```

Comment:
That no longer works because I deleted the style module. The style module isn't needed for any styles and using a non-existant class just for testing purposes (as it was before) is a pretty terrible hack. If we want to keep doing that then we should switch to some library that is designed for ephemeral test identifiers.